install/scripts/install-json-hpp.sh

```diff
@@ -1,21 +1,43 @@
 #!/bin/bash
 
+# Source fossology config file, for Proxy settings
+conf_file=/etc/fossology.conf
```

Comment:
It is not a good idea to hard code path here.
If you are installing FOSSology from Debian packages, then this path is correct.
If you install FOSSology from source, the path is more like `/usr/local/etc/fossology/fossology.conf`

Reply:
You can try the process target here:
https://github.com/fossology/fossology/blob/master/Makefile.process#L16
And make something like:
https://github.com/fossology/fossology/blob/master/install/defconf/fossology.conf.in#L48

The variables which you can look into are:
https://github.com/fossology/fossology/blob/master/Makefile.conf#L40

Reply:
Many thanks for the explanations and pointers - I have removed the proxy part of the PR altogether, I will have to work on this separately later.